source/docs/advanced/theme/analytics.md

```diff
@@ -199,7 +199,7 @@ However, please note that some integrations are not up to date. Even if you find
 
 ### Main integrations
 
-Here is a list of integrations frequently used accross eCommerce shops:
+Here is a list of integrations frequently used across e-commerce shops:
```

Comment:
This one is touchy :)

We got the discussions several times over the year and decided to not care.
I'm guilty of all the eCommerce as we first decided this (and took the habit).

Reply:
ahah 
and for the whole story, the spellchecker did not like it :)